students/itsabhi/Lesson3/list_lab.py

```diff
@@ -0,0 +1,88 @@
+#! /usr/bin/env Python3
+
+
+def series1():
+    my_list = ["Apples", "Pears", "Oranges", "Peaches"]
+    print(my_list)
+    add_fruit = input("Add another fruit: ")
+    my_list.append(add_fruit)
+    print(my_list)
+    (number) = input("Enter a number: ")
```

Comment:
I wouldn't put parentheses around `number` here, since parentheses have meaning in Python this could lead to confusion

Reply:
good catch!